src/muse/outputs/mca.py

```diff
@@ -918,11 +918,14 @@ def sector_lcoe(sector: AbstractSector, market: xr.Dataset, **kwargs) -> pd.Data
             prices = agent_market["prices"].sel(commodity=techs.commodity)
             demand = agent_market.consumption.sel(commodity=included)
             capacity = agent.filter_input(capacity_to_service_demand(demand, techs))
-            production = capacity * techs.fixed_outputs * techs.utilization_factor
-            production = convert_timeslice(
-                production,
-                demand.timeslice,
-                QuantityType.EXTENSIVE,
+            production = (
+                capacity
+                * convert_timeslice(
+                    techs.fixed_outputs,
+                    demand.timeslice,
```

Comment:
I think this is what confuses me the most about timeslices, specially now that you have shed some light into it: when do you use the timeslice of one particular array, like here, and when the global TIMESLICE? If they are the same I'd use always the global one, for clarity. And if they are not... why they are not and how can we know?

Reply:
I'm pretty sure these are all equivalent to `TIMESLICE` (or at least they should be), so there's no reason not to use the global. This is part of what I'm doing in #519, although that PR has become a bit too big so I might try to break this specific change into its own PR

Reply:
Actually it's more complicated than this. TIMESLICE is the global timeslicing scheme from the `timeslices` section of the settings file. However, MUSE does allow you to have different timeslicing schemes for different sectors (see [here](https://muse-os.readthedocs.io/en/latest/inputs/toml.html#standard-sectors)). I imagine this is so you can have less granularity in some sectors. For example, in the oil sector, you may only care about meeting demands at the seasonal level, so your timeslices might be "winter", "summer" etc. rather than "winter.weekend.morming", "winter.weekday.night" etc. In this case, maybe the timeslicing of the array isn't going to match up with TIMESLICE. Not sure whether it actually works like that though...

In that case, #519 is a waste of time as it gets rid of this functionality, although I do still want to tidy up how timeslices are dealt with as it's a complete mess at the moment.